packages/modules/manage-versions/src/components/CreateUpdateVersion/VersionModal.test.tsx

```diff
@@ -85,7 +85,7 @@ describe("VersionModal", () => {
       .getByText("Action")
       .closest("button") as HTMLButtonElement;
     expect(actionButton).not.toBeUndefined();
-    expect(actionButton.disabled).toBe(true);
+    expect(actionButton.getAttribute("aria-disabled")).toBe("true");
```

Comment:
why switched to checking aria attribute?

Reply:
itwinui now uses aria-disabled instead of disabled attribute